CommonTools/PileupAlgos/src/PuppiContainer.cc

```diff
@@ -92,10 +88,11 @@ double PuppiContainer::var_within_R(int iId, const vector<PseudoJet> & particles
 
     vector<double > near_dR2s;     near_dR2s.reserve(std::min(50UL, particles.size()));
     vector<double > near_pts;      near_pts.reserve(std::min(50UL, particles.size()));
+    const double R2 = R*R;
```

Comment:
start local variables and data members with lower case letter. Capitalization is reserved for class names.